scripts/deploy.sh

```diff
@@ -113,8 +113,13 @@ build_and_deploy() {
     printf '\n'
     printf 'Please review the build output now---run:\n'
     printf '    cd "%s" && python -m SimpleHTTPServer\n' "${preview_dir}"
-    printf 'Do you want to deploy? yes/no> '
-    read -r line
+    while true; do
```

Comment:
Alternately,

```shell
    line=
    while ! [ "${line}" = yes ] || [ "${line}" = no ]; do
        printf 'Do you want to deploy? yes/no> '
        read -r line
    done
```

which is more readable IMO.

Reply:
Done (in modified form), PTAL